src/rules/no-named-default.js

```diff
@@ -0,0 +1,32 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    function checkSpecifiers(type, node) {
+      if (node.source == null) return
+
+      const hasImportSpecifier = node.specifiers.some(function (im) {
+        return im.type === type
+      })
+
+      if (!hasImportSpecifier) {
+        return
+      }
+
+      node.specifiers.forEach(function (im) {
+        if (im.type !== type) return
+
+        if (im.imported.name === 'default') {
+          context.report(im.local,
+            'Using name \'' + im.local.name +
```

Comment:
The error is currently: "Using name `bar` as identifier for default export".
Technically, if you do `import bar from './foo'`, `bar` is also the name of the default export, but that's valid.

I'd change the error message to "Use default import syntax to import `bar`".